commands/bridge_push.go

```diff
@@ -32,20 +33,46 @@ func runBridgePush(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	parentCtx := context.Background()
+	ctx, cancel := context.WithCancel(parentCtx)
+	defer cancel()
+
+	done := make(chan struct{}, 1)
+
+	interrupt.RegisterCleaner(func() error {
+		fmt.Println("Received interrupt signal, stopping the import...\n(Hit ctrl-c again to kill the process.)")
```

Comment:
you need to implement the kill here as well

Reply:
done !